### PR TITLE
Add CI check for consistency on the Consensus changelog management

### PR DESCRIPTION
**Pull request by Jasagredo**
# Description

This CI check ensures that the management of the consensus versions and changelogs is consistent with the [release process](./ouroboros-consensus/docs/RealeaseProcess.md).

# Checklist

- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Commits have useful messages
    - [x] The documentation has been properly updated
    - [x] New tests are added if needed and existing tests are updated
    - [x] Any changes affecting Consensus packages must have an entry in the appropriate `changelog.d` directory created using [`scriv`](https://github.com/input-output-hk/scriv). If in doubt, see the [Consensus release process](../ouroboros-consensus/docs/ReleaseProcess.md).
    - [x] If this branch changes Network and has any consequences for downstream repositories or end users, said changes must be documented in [`interface-CHANGELOG.md`](../docs/interface-CHANGELOG.md)
    - [x] If serialization changes, user-facing consequences (e.g. replay from genesis) are confirmed to be intentional.
- Pull Request
    - [x] Self-reviewed the diff
    - [x] Useful pull request description at least containing the following information:
      - What does this PR change?
      - Why these changes were needed?
      - How does this affect downstream repositories and/or end-users?
      - Which ticket does this PR close (if any)? If it does, is it [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)?
    - [x] Reviewer requested

### CODE REVIEW

**Review by dnadales — COMMENTED** [review]
Nice. 

Could you please add a comment on the script that describes what it does? Also it might be a good idea to comment on what each step does.

**Comment by Jasagredo** [comment]
I added a description at the top and related it to each one of the steps @dnadales

**Comment by jorisdral** [comment]
Cool :smile: -- maybe we should record somewhere that this CI script should be updated when new packages are included, or when existing packages change names?

**Review by dnadales — APPROVED** [review]

File: scripts/ci/check-consensus-release.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                exit 1
+            else
+                echo "ouroboros-consensus version succesfully updated"
+            fi
+        fi
+    fi
+fi
+
+
+if [ -z "$cardano_versions" ]; then # See (1) above
```
Comment (abailly-iohk) [inline scripts/ci/check-consensus-release.sh:42]: Suggestion: Make a function and call it with 2 arguments? I know the usual advice for applying DRY is at the 3rd occurence, feel free to ignore my suggestion :)

**Review by abailly-iohk — APPROVED** [review]

**Pull request merged**